libcst/metadata/scope_provider.py

```diff
@@ -536,27 +535,49 @@ def accesses(self) -> Accesses:
         return Accesses(self._accesses)
 
 
+class BuiltinScope(Scope):
+    def __init__(self, globals: Scope) -> None:
+        self.globals: Scope = globals
+        super().__init__(parent=self)
+        for name in dir(builtins):
+            self._assignments[name].add(BuiltinAssignment(name, self))
+
+    def __contains__(self, name: str) -> bool:
+        return name in self._assignments and len(self._assignments[name]) > 0
```

Comment:
This should be enough since we never add more assignments?

```suggestion
        return name in self._assignments
```

Reply:
SGTM